mne/viz/circle.py

```diff
@@ -222,6 +222,25 @@ def plot_connectivity_circle(con, node_names, indices=None, n_lines=None,
     If ``facecolor`` is not set via :func:`matplotlib.pyplot.savefig`, the
     figure labels, title, and legend may be cut off in the output figure.
     """
+    return _plot_connectivity_circle(
+        con, node_names, indices, n_lines, node_angles, node_width,
+        node_colors, facecolor, textcolor, node_edgecolor, linewidth, colormap,
+        vmin, vmax, colorbar, title, colorbar_size, colorbar_pos,
+        fontsize_title, fontsize_names, fontsize_colorbar, padding,
+        fig, subplot, interactive, node_linewidth, show)
+
+
+def _plot_connectivity_circle(con, node_names, indices=None, n_lines=None,
```

Comment:
@adam2392 @agramfort I decided to go the private route -- it seemed weird to keep `plot_connectivity_circle` in `mne` rather than `mne-connectivity`. Ideally at some point we should make it so that this code is not duplicated with `mne-connectivity`, maybe by having `mne-connectivity.viz.plot_connectivity_circle` call this private function for example